dcat/index.html

```diff
@@ -1482,7 +1494,427 @@ <h4>Property: is referenced by</h4>
+<td>
+<p>This property is meant to be used to specify a version chain, consisting of snapshots of a resource.</p>
+<p>The notion of version used by this property is limited to versions resulting from revisions occurring to a resource as part of its life-cycle.</p>
+</td>
+</tr>
+</tbody>
+</table>
+
+<p>For guidance on the use of this property, see <a href="#version-history"></a>.</p>
+
+</section>
+
+<section id="Property:resource_has_version">
+
+<h4>Property: has version</h4>
+
+<aside class="note">
+<p>Property added in DCAT 3.</p>
+</aside>
+
+<table class="definition">
+<thead>
+<tr>
+<th>RDF Property:</th>
+<th><a href="http://www.w3.org/ns/dcat#hasVersion">dcat:hasVersion</a></th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td class="prop">Definition:</td>
+<td>This resource has a more specific, versioned resource [[?PAV]].</td>
+</tr>
+<tr>
+<td class="prop">Equivalent property:</td>
+<td><a data-cite="?PAV#d4e395"><code title="http://purl.org/pav/hasVersion">pav:hasVersion</code></a></td>
+</tr>
+<tr>
+<td class="prop">Sub-property of:</td>
+<td><a data-cite="?DCTERMS#http://purl.org/dc/terms/hasVersion"><code title="http://purl.org/dc/terms/hasVersion">dct:hasVersion</code></a></td>
+</tr>
+<td class="prop">Sub-property of:</td>
+<td><a data-cite="?PROV-O#generalizationOf"><code title="http://www.w3.org/ns/prov#generalizationOf">prov:generalizationOf</code></a></td>
+</tr>
+<tr>
+<td class="prop">Usage note:</td>
+<td>
+<p>This property is intended for relating a non-versioned or abstract resource to several versioned resources, e.g. snapshots [[?PAV]].</p>
+<p>The notion of version used by this property is limited to versions resulting from revisions occurring to a resource as part of its life-cycle. Therefore, its semantics is more specific than its super-property <a data-cite="?DCTERMS#http://purl.org/dc/terms/hasVersion"><code title="http://purl.org/dc/terms/hasVersion">dct:hasVersion</code></a>, which makes use of a broader notion of version, including editions and adaptations.</p>
+</td>
+</tr>
+</tbody>
+</table>
+
+<p>For guidance on the use of this property, see <a href="#version-history"></a>.</p>
+
+</section>
+
+<section id="Property:resource_is_version_of">
+
+<h4>Property: is version of</h4>
+
+<aside class="note">
+<p>Property added in DCAT 3.</p>
+</aside>
+
+<table class="definition">
+<thead>
+<tr>
+<th>RDF Property:</th>
+<th><a href="http://www.w3.org/ns/dcat#isVersionOf">dcat:isVersionOf</a></th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td class="prop">Definition:</td>
+<td>This resource is a version of a non-versioned or abstract resource.</td>
+</tr>
+<tr>
+<td class="prop">Inverse property of:</td>
+<td><a href="#Property:resource_has_version"><code title="http://www.w3.org/dcat#hasVersion">dcat:hasVersion</code></a></td>
+</tr>
+<tr>
+<td class="prop">Sub-property of:</td>
+<td><a data-cite="?DCTERMS#http://purl.org/dc/terms/isVersionOf"><code title="http://purl.org/dc/terms/isVersionOf">dct:isVersionOf</code></a></td>
```

Comment:
This link via data-cite gives 404. Could we use a plain href instead?
```suggestion
<td><a href="http://purl.org/dc/terms/isVersionOf"><code title="http://purl.org/dc/terms/isVersionOf">dct:isVersionOf</code></a></td>
```